doc/news/_preparation_next_release.md

```diff
@@ -283,9 +283,9 @@ This section keeps you up-to-date with the multi-language support provided by El
 - <<TODO>>
 - <<TODO>>
 
-### <<Binding>>
+### xfconf
 
-- <<TODO>>
+- Add xfconf binding. _(Richard Stöckl @Eiskasten)_
```

Comment:
Please describe more what it does etc. (Suitable for later highlight).

Reply:
Should be done by 6506d0e23efa25047381ef0cd2afa5aed13da3f7 or before